releasenotes/releasenotes_v0.8.0-alpha.md

```diff
@@ -0,0 +1,16 @@
+# Release Notes 0.2.0
```

Comment:
Please change to #Release Notes 0.8.0-alpha

Reply:
done